feature/login/src/main/java/com/hankki/feature/login/LoginRoute.kt

```diff
@@ -102,3 +116,15 @@ fun LoginScreen(
         )
     }
 }
+
+private fun startKakaoTalkLogin(context: Context, viewModel: LoginViewModel) {
+    UserApiClient.instance.loginWithKakaoTalk(context) { token, error ->
+        viewModel.handleLoginResult(token, error)
+    }
+}
```

Comment:
이렇게 람다로 바꿔서 사용하는건 어떨까요??
매개변수로 viewModel을 넘기는 것보다 이렇게 람다로 하는게 결합도를 더 낮출 수 있습니다아
그리고 함수의 로그인 로직과 결과 처리를 분리할 수도 있지요

```suggestion
private fun startKakaoTalkLogin(
    context: Context,
    handleLogin: (OAuthToken?, Throwable?) -> Unit,
) {
    UserApiClient.instance.loginWithKakaoTalk(context) { token, error ->
        handleLogin(token, error)
    }
}
```

Reply:
오..! 결합도까지 미처 생각을 못했네요😅
오늘도 열심히 배워갑니다.. ㅎㅎ
반영하였습니다! 감사합니다~